new pages (#5040)
sourcegraph/handbook: content/departments/people-talent/people-ops/process/how-we-engage-talent-outside-the-us/deel.md

# Deel - PEO/EOR employment

[Deel](https://www.deel.com/) is a privately-held payroll and compliance provider. That means anyone, anywhere can get hired and paid with Deel. Using a tech-enabled self-serve process, companies can legally hire full-time employees or independent contractors across the world. We use Deel to hire international employees and contractors. We refer to Deel and similar providers as Professional Employer Organizations (PEOs) or Employer of Record (EORs).

Deel acts as the intermediary between you (the employee) and the company you're working for. Deel engages with both parties using separate agreements. While you perform your duties at the company, you get hired and paid by Deel.

> ## How to get employed via a PEO/EOR?

Reviewer: ```suggestion
## How can I get employed via a PEO/EOR?
```

> During your interview process, your recruiter will explain the different types of employment available. Should you choose to come in on a PEO/EOR, they will ask you for certain details and then notify you as soon as the request has been approved.

Reviewer: ```suggestion
During your interview process, your recruiter will explain the different types of employment available. Should you prefer to come in on a PEO/EOR, they will ask you for certain details and then notify you as soon as the request has been approved.
```

After the People Ops team and the Finance team set everything up for you on the backend, Deel will contact you, notify you and request your signature. You can read about the onboarding task [here](https://help.letsdeel.com/hc/en-gb/articles/4414311737745-What-will-my-onboarding-process-look-like-).

### Point of contact

The recruiter will let you know that Deel will contact you.

> You will be assigned a Deel Onboarding Specialist. They will ask you to create an profile in Deel and send over compliance documentation. All of this must be done before your start date, you won’t be able to start until this is done. Read about [how to sign up as an employee](https://help.letsdeel.com/hc/en-gb/articles/5779651597585-How-to-sign-up-as-an-Employee) here.

Reviewer: ```suggestion
You will be assigned a Deel Onboarding Specialist. They will ask you to create an profile in Deel and send over compliance documentation. All of this must be done before your start date. You won’t be able to start until this is done. Read about [how to sign up as an employee](https://help.letsdeel.com/hc/en-gb/articles/5779651597585-How-to-sign-up-as-an-Employee) here.
```

If there are any delays, Deel will notify the People Ops team and propose a new start date. Your point of contact for any questions regarding employment will be answered by the Onboarding Specialist.

## FAQs

#### Why is there no mention of equity or stocks in my contract?

> Since your employment contract is with Deel, any equity grants or stock options offered by your company will need a separate agreement between you and them.

Reviewer: ```suggestion
Since your employment contract is with Deel, any equity grants or stock options offered by your company will need a separate agreement between you and your company.
```

We usa Carta as our default app for tracking options, please refer to [this handbook](../../../../../benefits-pay-perks/pay-expenses/compensation/index.md#components-of-compensation) page for more information. You can also confirm your number of options and grant date in BambooHR.

#### What’s the name of my employer?

You can check who employs you in [this page](https://help.letsdeel.com/hc/en-gb/articles/4414100793233-What-s-the-name-of-my-employer-).

#### How often does the exchange rate get updated for compensation?

> The exchange rate is based on a 12 month trailing average, but and it's only updated when there is a change to compensation - for example, Merit or Promotion or Band adjustment.

Reviewer: ```suggestion
The amount specified as compensation in your Deel contract is based on an exchange rate calculated based on the 12-month trailing average. Exchange rates are updated when there is a change to compensation, such as a Merit or Promotion or Band adjustment.
```

#### What does the onboarding process look like?

> Do to [this page](https://help.letsdeel.com/hc/en-gb/sections/8266345839249-Onboarding-Guide) to read about the process of being onboarded in Deel.

Reviewer: ```suggestion
Learn about the [process of being onboarded in Deel](https://help.letsdeel.com/hc/en-gb/sections/8266345839249-Onboarding-Guide).
```

#### How can I submit an expense in Deel?

> You can see this information in [this handbook page](../../../../../benefits-pay-perks/pay-expenses/expenses/expenses-through-deel.md).You can find more information on how to submit expenses in Deel in[ this page](https://help.letsdeel.com/hc/en-gb/sections/8266431839377-Expenses).

Reviewer: ```suggestion
You can see this information in [this handbook page](../../../../../benefits-pay-perks/pay-expenses/expenses/expenses-through-deel.md). You can find more information on how to submit expenses in Deel in [this page](https://help.letsdeel.com/hc/en-gb/sections/8266431839377-Expenses).
```

> #### The PTO policie in my country of recidence is different from Sourcegraph’s PTO policy, which one should I follow?

Reviewer 2: ```suggestion
#### The PTO policy in my country of residence is different from Sourcegraph’s PTO policy, which one should I follow?
```

> #### The PTO policie in my country of recidence is different from Sourcegraph’s PTO policy, which one should I follow?

Reviewer: ```suggestion
#### The PTO policies in my country of residence are different from Sourcegraph’s PTO policy; which one should I follow?
```

Sourcegraph will honor the unlimited [PTO policy](../../../../../benefits-pay-perks/benefits-perks/time-off/index.md), but you will be required to take the minimum mandatory ones for your country in order for us to be compliant with Deel and your country’s laws.

#### How do I submit time off?

You can check [this article](https://help.letsdeel.com/hc/en-gb/articles/4409044013201-How-do-I-request-time-off-) on how to ask for time off through Deel’s platform.

Sourcegraph uses Roots to submit PTO. There is an integration set up with Roots and Deel, so you will be able to submit time off via Roots and it will automatically sync to Deel. For this to work, your Deel email will need to match your Roots email (your Sourcegraph email). Make sure you change your Deel email to your corporate one if you wish this integration to apply to you.

You can read more about submitting time off via [Roots here](../../../../../benefits-pay-perks/benefits-perks/time-off/submitting-time-off.md).

#### How can I get an Employment Verification Letter from Deel?

> Go to [this article ](https://help.letsdeel.com/hc/en-gb/articles/4906891050769-How-do-I-download-an-Employment-Verification-Letter-)for more information on this topic.

Reviewer 3: FAQ suggestion: If I move will I be able to switch countries and still use deel?

Reviewer 3: FAQ suggestion: Will I be able to change my contract relationship later on if needed? (from EOR to something else?)

Author: Amazing! TY
